Gemfile

```diff
@@ -36,6 +36,9 @@ gem "acts-as-taggable-on"
 gem "ancestry", "~> 2.1"
 gem "bootsnap", ">= 1.1.0", require: false
 
+gem "graphiql-rails"
+gem "graphql"
```

Comment:
Neu hinzugefügte Gems gerne in entsprechender Comminachricht verlinken, sodass deren Quelle jederzeit schnell erreichbar ist.

> Add graphql gem to project
> * graphql gem creates a graphql server
> * graphiql creates a browser graphql client where queries can be run against the server

Sind diese beiden, richtig?

gem "graphiql-rails": https://github.com/rmosolgo/graphiql-rails
gem "graphql": https://github.com/rmosolgo/graphql-ruby

Reply:
Ja

Reply:
Hättest du Lust, das in die entsprechende Commitnachricht zu rebasen?